ts/Core/Chart/ChartDefaults.ts

```diff
@@ -0,0 +1,6 @@
+import type ChartOptions from './ChartOptions';
+
+const chartDefaults: ChartOptions = {
+};
+
+export default chartDefaults;
```

Comment:
If you do not move the `chart: { … }` default from `Options.ts` into this file, you can delete `ChartDefaults.ts`.